pandas/core/generic.py

```diff
@@ -913,20 +913,26 @@ def f(x):
     rename.__doc__ = _shared_docs['rename']
 
     def rename_axis(self, mapper, axis=0, copy=True, inplace=False):
-        """Alter the name of the index or columns.
+        """
+        Alter the name of the index or columns.
+
+        This function alters the name of the index or columns.
 
         Parameters
         ----------
         mapper : scalar, list-like, optional
             Value to set the axis name attribute.
-        axis : int or string, default 0
+        axis : int, str, default 0
+            The index or the name of the axis.
```

Comment:
Where inconsistent with axis. But

            axis : {0 or 'index', 1 or 'columns'}, default 0
seems to be the most common.

Reply:
Done